### PR TITLE
Add attributes for the RHEL7 inspec checks

### PR DESCRIPTION
**Pull request by lihkin213**
Need to pull in the attributes for the RHEL7 inspec check. We had the attributes in the defaults but were not pulling them in the attributes file

### CODE REVIEW

**Comment by nirajdp76** [comment]
ok to test

**Comment by edtubillara** [comment]
LGTM

File: roles/inspec/templates/etc/inspec/host-controls/attributes.yml [hunk 1]
```diff
@@ -1,3 +1,6 @@
 {% for name,value in inspec.openstack.horizon.attributes.iteritems() %}
 {{ name }}: {{ value }}
 {% endfor %}
+{% for name,value in inspec.rhel7.attributes.iteritems() %}
```
Comment (nirajdp76) [inline roles/inspec/templates/etc/inspec/host-controls/attributes.yml]: what will happen on ubuntu?

Comment (lihkin213) [inline roles/inspec/templates/etc/inspec/host-controls/attributes.yml]: Currently we do not have support for Ubuntu Inspec checks, but we will add the attributes for it when we have that in place.

Comment (nirajdp76) [inline roles/inspec/templates/etc/inspec/host-controls/attributes.yml]: we should set role default to false and ensure rhosp/centos envs have inspec.enabled: true

Comment (paulczar) [inline roles/inspec/templates/etc/inspec/host-controls/attributes.yml]: agreed!   although better might be a check for the OS and only add it in for rhel7 inside the template itself.

Comment (lihkin213) [inline roles/inspec/templates/etc/inspec/host-controls/attributes.yml]: So, I am going to keep the structure of how the enablement is done the way it is. So role defaults enabled: True and the defaults2.0.yml enabled: False. This is consistent with what we did for serverspec. 
Added in checks where we do not putt down the inspec-stigs-rhel7 repo on Ubuntu

**Comment by lihkin213** [comment]
Retest

**Comment by lihkin213** [comment]
retest

**Comment by lihkin213** [comment]
retest

**Comment by lihkin213** [comment]
retest

**Comment by nirajdp76** [comment]
ok to test

**Comment by nirajdp76** [comment]
retest

File: roles/inspec/defaults/main.yml [hunk 2]
```diff
@@ -12,6 +12,7 @@ inspec:
     - name: inspec-stig-rhel7
       version: master
       url: https://github.com/inspec-stigs/inspec-stig-rhel7/archive/master.tar.gz
+      enabled: "{{ ursula_os == 'rhel' }}"
```
Comment (paulczar) [inline roles/inspec/defaults/main.yml]: I would rather see this be simply `true|false`

File: roles/inspec/templates/etc/inspec/host-controls/attributes.yml [hunk 3]
```diff
@@ -1,3 +1,8 @@
 {% for name,value in inspec.openstack.horizon.attributes.iteritems() %}
 {{ name }}: {{ value }}
 {% endfor %}
+{% if ursula_os == 'rhel' %}
```
Comment (paulczar) [inline roles/inspec/templates/etc/inspec/host-controls/attributes.yml]: I'm concerned that this is not based off `enabled: "{{ ursula_os == 'rhel' }}"`

**Comment by lihkin213** [comment]
Closing this out as the changes here will be handled by https://github.com/blueboxgroup/ursula/pull/2588

**Pull request closed without merging**